.github/workflows/templates-api.yml

```diff
@@ -24,28 +25,70 @@ jobs:
         uses: actions/checkout@v3
         with:
           ref: ${{ github.event.pull_request.head.ref }}
+
+      - name: Create artifacts directory
+        run: mkdir -p /tmp/artifacts
+
+      - name: Install gron tool
+        run: |
+          url="https://github.com/tomnomnom/gron/releases/download/v0.7.1/gron-linux-amd64-0.7.1.tgz"
+          curl -L "$url" | tar -xz -C /usr/local/bin
+
       - name: Start MiniKube
         run: ./start_minikube.sh
+
       - name: Set Kubernetes namespace
         run: kubectl config set-context --current "--namespace=$KUBERNETES_NAMESPACE"
+
       - name: Checkout BASE branch
         uses: actions/checkout@v3
         with:
           ref: ${{ github.event.pull_request.base.ref }}
-      - name: Deploy the previous version, from the BASE branch
+
+      - name: Deploy the old version, from the BASE branch
+        continue-on-error: true
```

Comment:
Pri prvom PR pre `Buffer API` tam nebude old state, tak som tu dal `continue-on-error: true`.
Moze sa to hodit aj v nejakych inych situaciach, ked bude old state rozbity.